src/fourcipp/fourc_input.py

```diff
@@ -65,6 +66,42 @@ def is_section_known(section_name: str) -> bool:
 CONVERTER = Converter()
 
 
+def modify_input_with_defaults(input_path: pathlib.Path, overwrite_flag: bool) -> None:
```

Comment:
Having the same name as in the CLI reduces a bit of code.
```suggestion
def modify_input_with_defaults(input_file: pathlib.Path, overwrite: bool) -> None:
```

Reply:
I don't see the benefit (reduction of code). I use the term *_path, since it is a pathlib variable.